lib/modules/apostrophe-dialog-box-apos-rich-text-widgets/index.js

```diff
@@ -1,4 +1,19 @@
 module.exports = {
   improve: 'apostrophe-rich-text-widgets',
-  dialogBox: false
```

Comment:
I think you'd still disable improving the rich text widget as the rich text widget does not have an interface to interact with schemas (essentially bypassing them).

Does the adding join fields to the widget play into the way CKEditor is making contact w apostrophe?